plugins/azure-devops-backend/api-report.md

```diff
@@ -19,12 +20,24 @@ import { WebApi } from 'azure-devops-node-api';
 export class AzureDevOpsApi {
   constructor(logger: Logger_2, webApi: WebApi);
   // (undocumented)
+  getBuildDefinitions(
+    projectName: string,
+    definitionName: string,
+  ): Promise<BuildDefinitionReference[]>;
+  // (undocumented)
   getBuildList(
     projectName: string,
-    repoId: string,
```

Comment:
@awanlin this looks like a breaking change here right?

Reply:
Yes, sorry for the slow reply. I'd like to refactor this, I think there is a non-break approach that I'd like to take that will also improve a few things

Reply:
👍 , it's still breaking because of the additions of required methods in the API, but the impact is ofc smaller. Best make it a `minor` change tho

Reply:
I've updated this to `minor` now, thanks 👍